src/window.rs

```diff
@@ -628,6 +628,8 @@ impl Window {
     ///
     /// - **iOS:** Can only be called on the main thread.
     /// - **Android:** Will always return `None`.
+    /// - **Wayland:** Can return `Borderless(None)` when no monitors are presented.
```

Comment:
What do you mean by "no monitors are presented"? Like no monitors are *present*? Usually a monitor does the presenting and isn't presented, right?

Reply:
It means that user powered off every monitor it had, however our window was fullscreened when the user done that, so it's fullscreened, but there are no outputs.